capabilities/weather/current-city/maps/current-city.ts

```diff
@@ -1,40 +1,30 @@
-/* eslint-disable @typescript-eslint/no-unsafe-member-access */
-/* eslint-disable jest/no-try-expect */
-/* eslint-disable jest/no-conditional-expect */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
 /* eslint-disable jest/no-export */
 
 import { describe, expect } from '@jest/globals';
-import { Provider } from '@superfaceai/one-sdk';
-
-import {
-  SuperfaceClient,
-  WeatherCurrentCityProfile,
-} from '../../../../superface/sdk';
+import { SuperfaceTest } from '@superfaceai/testing-lib';
 
 export const currentCityTest = (providerName: string): void => {
   describe(`weather/current-city/${providerName}`, () => {
-    let client: InstanceType<typeof SuperfaceClient>;
-    let profile: WeatherCurrentCityProfile;
-    let provider: Provider;
-
-    beforeEach(async () => {
-      client = new SuperfaceClient();
-      profile = await client.getProfile('weather/current-city');
-      provider = await client.getProvider(providerName);
-    });
+    let superface: SuperfaceTest;
 
-    it('should find provider', async () => {
-      expect(provider).not.toBeUndefined();
+    beforeEach(() => {
+      superface = new SuperfaceTest();
```

Comment:
You can set up profile and provider here if there are no test runs with different profile or provider.

```suggestion
      superface = new SuperfaceTest({
        profile: 'weather/current-city',
        provider: providerName,
      });
```